packages/qtcreator.rb

```diff
@@ -56,23 +59,13 @@ def self.install
     system "echo '    }' >> qt-installer-script.qs"
     system "echo '    gui.clickButton(buttons.FinishButton);' >> qt-installer-script.qs"
     system "echo '}' >> qt-installer-script.qs"
-    case ARCH
-    when 'i686'
-      system 'curl -#LO http://qtmirror.ics.com/pub/qtproject/archive/online_installers/2.0/qt-unified-linux-x86-2.0.5-2-online.run'
-      abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest(File.read('qt-unified-linux-x86-2.0.5-2-online.run')) == 'b9dfef211d3122ab2f1b1e96aa7e2357ebdeb068c750eeb423b9396a3f55d619'
-      system "install -Dm755 qt-unified-linux-x86-2.0.5-2-online.run #{CREW_PREFIX}/tmp/qt-unified-linux-x86-2.0.5-2-online.run"
-      system "#{CREW_PREFIX}/tmp/qt-unified-linux-x86-2.0.5-2-online.run --script qt-installer-script.qs"
-      system "rm -f #{CREW_PREFIX}/tmp/qt-unified-linux-x86-2.0.5-2-online.run"
-    when 'x86_64'
-      system 'curl -#LO http://qt.mirror.constant.com/archive/online_installers/3.2/qt-unified-linux-x64-3.2.3-online.run'
-      abort 'Checksum mismatch. :/ Try again.'.lightred unless Digest::SHA256.hexdigest(File.read('qt-unified-linux-x64-3.2.3-online.run')) == 'b724dabc0e7bfde1e078de3edcdd87c94b60031f9896d7119099a66191d75eec'
-      system "install -Dm755 qt-unified-linux-x64-3.2.3-online.run #{CREW_PREFIX}/tmp/qt-unified-linux-x64-3.2.3-online.run"
-      system "#{CREW_PREFIX}/tmp/qt-unified-linux-x64-3.2.3-online.run --script qt-installer-script.qs"
-      system "rm -f #{CREW_PREFIX}/tmp/qt-unified-linux-x64-3.2.3-online.run"
-    end
-    system "mkdir -p #{CREW_DEST_PREFIX}/bin"
-    system "mkdir -p #{CREW_DEST_PREFIX}/share"
-    system "cp -r #{CREW_PREFIX}/share/qt #{CREW_DEST_PREFIX}/share"
-    system "ln -s #{CREW_PREFIX}/share/qt/Tools/QtCreator/bin/qtcreator #{CREW_DEST_PREFIX}/bin/qtcreator"
+    downloader 'https://download.qt.io/archive/online_installers/4.6/qt-unified-linux-x64-4.6.1-online.run', '11f8e9307a31d283b7ad80050e9d43c1e94dbc2ea283eb6fa9d051549f2bbccb'
+    FileUtils.install 'qt-unified-linux-x64-4.6.1-online.run', "#{CREW_PREFIX}/tmp/qt-unified-linux-x64-4.6.1-online.run", mode: 0o755
+    system "#{CREW_PREFIX}/tmp/qt-unified-linux-x64-4.6.1-online.run --script qt-installer-script.qs"
+    FileUtils.rm_f "#{CREW_PREFIX}/tmp/qt-unified-linux-x64-4.6.1-online.run"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/bin"
+    FileUtils.mkdir_p "#{CREW_DEST_PREFIX}/share"
+    FileUtils.cp_r "#{CREW_PREFIX}/share/", "#{CREW_DEST_PREFIX}/share"
```

Comment:
Won't this result in `#{CREW_DEST_PREFIX}/share/share`?

Reply:
Line 67 is not needed with `FileUtils.cp_r "#{CREW_PREFIX}/share/", CREW_DEST_PREFIX.to_s`?

Reply:
done!